src/Carnac.Logic/KeyProvider.cs

```diff
@@ -101,10 +106,19 @@ KeyPress ToCarnacKeyPress(InterceptKeyEventArgs interceptKeyEventArgs)
                            interceptKeyEventArgs.Key <= Keys.Z;
 
             var inputs = ToInputs(isLetter, winKeyPressed, interceptKeyEventArgs);
+            try
```

Comment:
`config.ShowApplicationIcon` should come into effect at this point as well I think. Thoughts?